src/theory/bv/bv_subtheory_core.cpp

```diff
@@ -566,7 +567,7 @@ bool CoreSolver::doExtfInferences(std::vector<Node>& terms)
                               nm->mkNode(kind::LT, n, max));
         Trace("bv-extf-lemma")
             << "BV extf lemma (range) : " << lem << std::endl;
-        d_bv->getOutputChannel().lemma(lem);
+        d_bv->d_inferManager.lemma(lem);
```

Comment:
You should use `d_inferManager` from the base class here (there are several other similar places like this).  I would not reference TheoryBV's inference manager like this since where this lives is potentially subject to change.